recipes/strainseeker/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "1.5" %}
+
+package:
+  name: strainseeker
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/jiangweiyao/StrainSeeker/archive/master.tar.gz
```

Comment:
Can you please create a release and use the release tarball. We try to avoid to use the master branch, as this is changing frequently.

Reply:
It is changed and the new recipe passed the check. Please let me know if there is anything else. Thank you!